src/index.ts

```diff
@@ -20,7 +20,11 @@ export = (app: Application): void => {
       "pull_request.reopened",
       "pull_request.synchronize",
     ],
-    async (context) => {
+    // TODO(tianhaoz95): change this to strongly typed webhook
+    // event to make it safer.
+    /* eslint-disable */
+    async (context: Context<any>) => {
```

Comment:
Codacy found an issue: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6311563)